src/array_deque.rs

```diff
@@ -34,7 +34,7 @@ impl<const N: usize> Meta for ArrayMeta<N> {
 /// All values are stored inline; that is, the size of of `ArrayDeque<T, N>` is
 /// *at least* `size_of::<[T; N]>()`, regardless of the number of elements
 /// currently stored in the deque.
-#[derive(Clone, Debug)]
+#[derive(Clone, Copy, Debug)]
```

Comment:
I think it would be better to derive Copy only if T is Copy, like this:
```rust
impl<T: Copy + Default, const N: usize> Copy for ArrayDeque<T, N> {}
```
in order to be able to keep using this for non-copy types.

Reply:
Changed.